app/services/policies/policies_list_service.rb

```diff
@@ -6,11 +6,16 @@
 class Policies::PoliciesListService
   def self.call(account, builtin: true)
     list = PolicyList.new
-    if builtin
-      response = ::JSONClient.get(ThreeScale.config.sandbox_proxy.apicast_registry_url)
-      return unless response.ok?
-
-      list.merge!(PolicyList.from_hash(response.body['policies']))
+    apicast_registry_url = ThreeScale.config.sandbox_proxy.apicast_registry_url
+    if builtin && apicast_registry_url
+      begin
+        response = ::JSONClient.get(apicast_registry_url)
+        return unless response.ok?
+        list.merge!(PolicyList.from_hash(response.body['policies']))
+      rescue => error
```

Comment:
I don't think that rescuing without specifying the error is a good idea 🤔

Reply:
I don't know what can go wrong for this specific case. Network errors for sure, and I also don't know if `list.merge!(PolicyList.from_hash(response.body['policies']))` will go well if `response.body['policies']` is not present or it is not hash, and I don't even know if that is a possibility 🤷‍♀️

Reply:
Well if it is a status `:ok`, I expect the third-party service to send a valid body otherwise HTTP is going down :)

Reply:
So this could be a `fetch` method, 
You can rescue those specific errors 

https://github.com/nahi/httpclient/blob/0eda210a41bef76e44c7c0cb3020024e1150dfd4/lib/httpclient.rb#L253-L281